src/main/java/com/dev/config/WebConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.dev.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = {"com.dev.controllers"})
+public class WebConfig {
+
+    @Bean
+    public InternalResourceViewResolver viewResolver() {
```

Comment:
```suggestion
    public InternalResourceViewResolver getViewResolver() {
```
назва методу - дієслово або фраза на його основі(get, put, createNewFile)

Reply:
got it